client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -1337,12 +1337,23 @@
                         scope.saveDisabled = false;
                     });
                 };
-                keyboardManager.bind('ctrl+q', function (e) {
-                    e.preventDefault();
+                scope.$on('key:ctrl:shift:u', function($event, event) {
+                    event.preventDefault();
+                    if (scope.item._locked &&
```

Comment:
it would be good to fix this code duplication here and in view, mabe make this part of `can_unlock` function?